zk/watcher.go

```diff
@@ -142,13 +151,15 @@ func (w *Watcher) runHooks() error {
 	w.hooksLock.Lock()
 	defer w.hooksLock.Unlock()
 
+	log.Println("Recreating ephemeral nodes")
```

Comment:
Let's add a common prefix (`sequins ZK`?) to all logs from this file.

Reply:
OK. I talked to the team a few days ago I am missing log4j.